app/partials/desktop/confirm_modal.html

```diff
@@ -25,6 +25,15 @@
         <ng-pluralize count="count"
                      when="confirm_modal_X_messages_delete">
         </ng-pluralize>
+        <a class="tg_checkbox" ng-if="revocable" ng-click="data.revoke = !data.revoke" ng-class="data.revoke ? 'tg_checkbox_on' : ''">
```

Comment:
Oh, my bad here. I didn't try this on TDesktop, just iOS. 
But it also includes a new line on TDesktop, when removing in supergroup (which is always for everyone):
<img width="330" alt="screen shot 2017-04-03 at 18 58 24" src="https://cloud.githubusercontent.com/assets/775173/24618384/8d3a1a3c-189f-11e7-8f38-c8c5322dca6b.png">

Then 2 buttons are not required, checkbox is OK with me. Just add this label please.